css/index.css

```diff
@@ -143,4 +237,116 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+/*about*/
+header{
+    display: flex;
+    align-items: baseline;
+    justify-content: space-around;
+    margin-bottom: 20px;
+}
+
+header nav{
+    display: flex;
+    justify-content: space-between;
+    width: 500px;
+    margin-top: 60px;
+
+}
+
+header nav a{
+    text-decoration: none;
+    color: gray;
+    font-size: 15px;
+     display: flex;
+    justify-content: space-between;
+
+}
+header nav a:hover{
+    color: purple;
+    background-color: pink;
+    animation: pulse 5s infinite;
+  }
+  
+  @keyframes pulse {
+    0% {
+      background-color: purple;
+      color:pink;
+    }
+    100% {
+      background-color: #ffddf4;
+      color:black;
+    }
+  }
+
+}
+header .logo{
+    margin-right: 100px;
+    margin-top: 30px;
+}
+.top-content .home-top-img{
+    margin-bottom: 20px;
+    width: 100%;
+}
+.top-content-about {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+}
+.top-content .home-top-img-about{
+    margin-bottom: 20px;
+    width:100%;   
+}
+.top-content-about h2{
+    margin-top: 20px;
+    padding-left: 25px;
+    
+}
+
+.top-content-about p{
+    padding-left: 25px;
+    margin-bottom: 25px;
+}
+
+.middle-content-about{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    line-height: 1.2;
+}
+/*info cards*/
+.infoCard{
+    border-bottom: 2px dashed black;
+    width: 48%;
+    margin-bottom: 20px;
+    margin-right: 10px;
+    padding: 10px;
+}
+
+.infoCard .button{
+  border: 1px double black;
+  font-size: 16px;
+  background: white;
+  width: 170px;
+  height: 32px;
+  margin-top: 10px;
+  cursor: pointer;
+}
```

Comment:
The hover effect is not working because you dont have an a tag here. try using button:hover { styles }